operator/main.go

```diff
@@ -73,10 +68,15 @@ func main() {
 	logger := zap.New(zap.UseFlagOptions(&opts))
 	ctrl.SetLogger(logger)
 
+	watchNamespace := namespace
+	if clusterwide {
+		watchNamespace = ""
+	}
+	setupLog.Info("Starting manager", "clusterwide", clusterwide)
```

Comment:
It is slightly not clear to me why this is enough to enable `clusterwide` perhaps add a note with a description?
can we have `clusterwide` and per `namespace` install?

Reply:
If its namespaced then the controller won't watch for changes outside its namespace

Reply:
added note